.travis.yml

```diff
@@ -26,117 +31,101 @@ env:
         - NODE_ENV=test
         - NPM_CONFIG_PROGRESS=false
         - COZY_DESKTOP_HEARTBEAT=1000
-services:
-    - docker
-cache:
-    yarn: true
-    directories:
-        - $HOME/.cache/electron
-        - $HOME/.cache/electron-builder
-        - $HOME/bin
-addons:
-    apt:
-        sources:
-            - ubuntu-toolchain-r-test
-        packages:
-            - gcc-4.8
-            - g++-4.8
+          #- CXX=$([ "$TRAVIS_OS_NAME" = "linux" ] && echo "g++-4.8" || echo $CXX)
```

Comment:
I commented this variable definition since it made some tests fail in this parallel situation and actually doesn't seem to be useful but wanted to discuss its removal.

Reply:
I think you can delete this line